hpobench/benchmarks/surrogates/svm_benchmark.py

```diff
@@ -0,0 +1,199 @@
+        """
+        seed = seed if seed is not None else np.random.randint(1, 100000)
+        fidel_space = CS.ConfigurationSpace(seed=seed)
+
+        fidel_space.add_hyperparameters([
+            CS.UniformFloatHyperparameter("dataset_fraction", lower=0.0, upper=1.0, default_value=1.0, log=False)
+        ])
+        return fidel_space
+
+    @staticmethod
+    def get_meta_information():
+        """ Returns the meta information for the benchmark """
+        return {'name': 'ParamNet Benchmark',
+                'references': ['@InProceedings{falkner-icml-18,'
+                               'title       = {{BOHB}: Robust and Efficient Hyperparameter Optimization at Scale},'
+                               'url         = http://proceedings.mlr.press/v80/falkner18a.html'
+                               'author      = {Falkner, Stefan and Klein, Aaron and Hutter, Frank}, '
+                               'booktitle   = {Proceedings of the 35th International Conference on Machine Learning},'
+                               'pages       = {1436 - 1445},'
+                               'year        = {2018}}',
+                               '@inproceedings{klein2017fast,'
+                               'title       = {Fast bayesian optimization of machine learning hyperparameters on '
+                               '               large datasets},'
+                               'author      = {Klein, Aaron and Falkner, Stefan and Bartels, Simon and '
+                               '               Hennig, Philipp and Hutter, Frank},'
+                               'booktitle   = {Artificial Intelligence and Statistics},'
+                               'pages       = {528--536},'
+                               'year        = {2017},'
+                               'organization={PMLR}}'],
+                'code': 'https://github.com/automl/HPOlib1.5/blob/development/'
+                        'hpolib/benchmarks/surrogates/paramnet.py'
+                }
+
+    @staticmethod
+    def convert_config_to_array(configuration: Dict, fidelity: Dict) -> np.ndarray:
+        """
+        This function transforms a configuration to a numpy array.
+
+        Parameters
+        ----------
+        configuration : Dict
+        fidelity : Dict
+
+        Returns
+        -------
+        np.ndarray - The configuration transformed back to its original space
+        """
+        cfg_array = np.zeros(3)
+        cfg_array[0] = configuration['C']
+        cfg_array[1] = configuration['gamma']
+        cfg_array[2] = fidelity['dataset_fraction']
+        return cfg_array.reshape((1, -1))
+
+    @AbstractBenchmark.check_parameters
+    def objective_function(self, configuration: Union[CS.Configuration, Dict],
+                           fidelity: Union[CS.Configuration, Dict, None] = None,
+                           rng: Union[np.random.RandomState, int, None] = None, **kwargs) -> Dict:
+
+        if fidelity['dataset_fraction'] < self.s_min:
```

Comment:
What's the reason to not directly set the lower fidelity bound to 1/512 or 100 / 50000 since there is only one dataset?